src/components/Popup.js

```diff
@@ -17,6 +18,14 @@ export default class Popup {
     this.#removeHandleEscClose();
   }
 
+  changeButtonText (isLoading) {
+    if (isLoading) {
+      this.submitButton.textContent = "Saving...";
```

Comment:
And in `PopupDeleteCard.js` you'll need to make the text `Deleting...` and `Yes`